analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UnusedPrivateMember.cs

```diff
@@ -67,10 +67,11 @@ private void UnusedPrivateMember_MyEvent()
             get; set;
         }
         private void Method() { } // Noncompliant {{Remove the unused private method 'Method'.}}
+//                   ^^^^^^
         private class Class { }// Noncompliant {{Remove the unused private class 'Class'.}}
-//      ^^^^^^^^^^^^^^^^^^^^^^^
+//                    ^^^^^
         private struct Struct { }// Noncompliant {{Remove the unused private struct 'Struct'.}}
```

Comment:
for better test coverage, could you please add also a test for  unused `record` and `record struct`?

Reply:
Added them.